src/components/Footer/Aletheiainfoadress.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslation } from "next-i18next";
+import { useAppSelector } from "../../store/store";
+
+const Aletheiainfoadress = () => {
+  const { t } = useTranslation();
+  const { vw } = useAppSelector((state) => state);
+
+  return (
+    <>
+      <span
+        style={{
+          fontSize: "14px",
+          marginTop: "0px",
+          textAlign: "center",
```

Comment:
Suggestion: Create a styled component using the `styled-component` library allowing code reuse.